files/hiera.yaml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
I would just put in Compliance Engine and default/common here. This is just going to confuse people.

```
---
version: 5
defaults:
  datadir: data
  data_hash: yaml_data
hierarchy:
  - name: Common
    path: common.yaml
  - name: SIMP Compliance Engine
    lookup_key: compliance_markup::enforcement
```

Reply:
Removed all but common.yaml and referred users Puppet and SIMP Hiera documentation.